src/PharoThings-Devices-HCSR04.package/PotHCSR04Device.class/instance/readPinStateDuration.st

```diff
@@ -0,0 +1,7 @@
+controlling
+readPinStateDuration
+	| pulseStart pulseEnd |
+	echoPin beDigitalInput; enablePullDownResister.
```

Comment:
I think my removed comment is still relevant.
In my idea such pin configuration should be performed in device>>connect method. It will be done only once during installation in the board and it will allow to not think about it in any other places.

Reply:
I see your idea and I agree. But in this case, if it using 1 PIN to send and read, it needs to become input and output all the time. So to simplify the code, I did this a default behavior to 3 and 4 PINs models.